### DISCUSSION TITLE
I do not have access to COPILOT and I cannot upload my Your student ID again because it is locked.

### DISCUSSION THREAD

**Discussion by rvsweb** [community/community#86521] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

In my study center we were given the possibility to access the COPILOT extension of VSC.
The educational center created the student ID and signed it for us, all my colleagues were able to upload it without problem to the platform and can enjoy the use of the COPILOT extension but it does not allow me, I have tried to upload the file again with more quality, with a more appropriate size but it does not allow me to upload it again, I have sent a ticket to the platform to solve the problem but it does not solve the problem because it says to remove the image of the student ID that I uploaded and upload it again but for whatever reason the platform where the student ID is uploaded is blocked.
I can't send a ticket directly to solve the problem and the emails I send don't solve the problem.
Please someone help me, I need the COPILOT extension because we are programming artificial intelligence with Python and I need it to be able to develop very complicated artificial intelligence models, I am falling behind in my studies.
All my classmates and teacher can use it , we all did it from the same classroom , same computers and same network but it doesn't give me access.
Thank you for your attention 
![image](https://github.com/community/community/assets/8049648/48e99321-63e6-45c1-93ac-58422566522c)

**Comment by rvsweb** [comment, marked as answer]
I have already solved the problem by adding a new screenshot of my student card from the school along with another card from the school.
Very unintuitive error messages that do not help to solve the problem.

**Comment by FelixAshong** [comment]
Faced that recently
